### ISSUE TITLE
On selecting co-author: Failed to update input: Cannot read property 'inputBox' of undefined

### ISSUE THREAD

**Issue by anarchist912** [rkotze/git-mob-vs-code#61]
### Prerequisites

- [x] Checked that your issue isn't already filed
- [x] Tried upgrading to the latest git-mob-vs-code and git-mob version
- [x] Tried upgrading to the latest git version

### Description
Fresh install, vs code version 1.52.1. Clicking on a user to add them as co-authors results in dialog with error message:
```
Failed to update input: Cannot read property 'inputBox' of undefined
```

### Steps to Reproduce
see above

**Expected behaviour:** User gets added to selected co-authors.

**Actual behaviour:** Error message appears, user stays unselected.

**Reproduces how often:** Every time until now.

### Versions

- operating system and version: Ubuntu 19.10
- git-mob-vs-code version: v1.11.0
- git-mob version (`git-mob --version`): 1.1.0
- git version (`git --version`): 2.28.0

**Comment by rkotze** [comment]
@anarchist912 hi,

Does this issue happen when you have multiple repositories open in one instance of VS Code? This is a bug in VS Code Git ext, where the extension API does not show which is selected. You can find more details here: https://github.com/rkotze/git-mob-vs-code/issues/48 

If just one repository then restart VS code or "reload window" command should solve the issue.

**Comment by anarchist912** [comment]
@rkotze hi! Sorry, yes, this is exactly the case. So this can be closed and I'll wait on #48. Your extension really becomes interesting within a multi-repo workspace. ;)
